reposilite-frontend/src/components/Dashboard.vue

```diff
@@ -32,4 +33,9 @@ export default {
         Menu
     }
 }
-</script>
\ No newline at end of file
+</script>
+
+<style lang="stylus" scoped>
+html
```

Comment:
You shouldn't style parent elemets. It's better to style root element of the template instead